eclipse-scout-core/src/form/lifecycle/FormLifecycle.ts

```diff
@@ -36,27 +37,24 @@ export class FormLifecycle<TValidationResult extends ValidationResult = Validati
   }
 
   override invalidElements(): { missingElements: TValidationResult[]; invalidElements: TValidationResult[] } {
-    let missingFields = [];
-    let invalidFields = [];
+    const missingElements = [],
+      invalidElements = [];
 
     this.widget.visitFields(field => {
       let result = field.getValidationResult();
       if (result.valid) {
         return result.visitResult;
       }
-      // error status has priority over mandatory
-      if (!result.validByErrorStatus) {
-        invalidFields.push(result);
-      } else if (!result.validByMandatory) {
-        missingFields.push(result);
+      // mandatory has priority over error status
```

Comment:
Discussed this with Claudio and we considered the following aspects:

1. an empty mandatory field results in an ERROR
2. a field with error status XYZ results in severity XYZ
3. the message of the error status is never shown in the validation result message box

Therefore an empty mandatory field with an additional error status needs to be added to

- `missingFields` if the error status has a severity less or equal than WARNING, otherwise the validation result can be ignored and the Form can be stored with an empty mandatory field
- `invalidFields` or `missingFields` if the error status is ERROR

The only difference with this change will be an empty mandatory field with error status ERROR being shown in the missing block and no longer in the invalid block, no information of the error status is lost as the message was never shown.
If one wants to keep the former priority it will result in more complicated code and therefore the result of our discussion was to change it.